aten/src/ATen/native/cuda/DistanceKernel.cu

```diff
@@ -325,7 +324,10 @@ void cdist_backward_kernel_impl(Tensor& result, const Tensor& grad, const Tensor
   const int64_t r1 = x1.size(-2);
   const int64_t r2 = x2.size(-2);
   const int64_t m = x1.size(-1);
-  const int64_t d = x1.size(0);
+  int64_t d = 1;
```

Comment:
Shorter as a ternary operator, but sometimes less readable: `int64_t d = x1.dim() > 2 ? 1 : x1.size(0)`.

What is `d`? It doesn't have a very descriptive name